sag_ji3.par

```diff
@@ -1,118 +1,118 @@
+-47.567830099549400	)\\!	405/416		-24
+-45.600475235757100	.\\!	38/39	._	-23
+-44.195221761619800	\\!	6400/6561	_	-22
+-41.665765508172500	'\\!	125/128	'_	-21
+-40.260512034035200	~!/	16384/16767		-20.8
+-39.698410644380200	(!(	44/45	a	-20
+-38.061940349778500	\!~	45/46		-19.9
+-37.871581128001700	.~!)	524288/535815		-19.6
+-37.309479738346700	'(!(	8019/8192	'a	-19
+-36.588490003137900	~!)	48/49		-18.4
+-35.118091464366500	.(!	49/50	.j	-18
+-34.358447442658300	(!	45056/45927	j	-17
+-32.391092578866000	!/	54/55	k	-16.6
+-31.407415146969900	'(!	55/56	'j	-16.4
+-30.985839104728600	)!)	56/57		-16
+-30.002161672832500	.!)	3584/3645	.t	-15
+-28.596908198695100	!)	63/64	t	-14
+-26.067451945247900	'!)	2066715/2097152	't	-13.4
+-24.885981585695500	)\!	40960/41553		-13
+-23.955448448755500	.\!	524288/531441	.\	-12.4
+-23.116419649559500	')\!	75/76	'	-12
+-22.694843607318300	\!	80/81	\	-11
+-20.305912701284800	)!~	19456/19683		-10.7
+-19.743811311629800	'\!	2025/2048	'\	-10
+-18.760133879733700	~~!	98/99		-9
+-17.354880405596300	!~	729/736	z	-8
+-15.387525541804000	~!(	4096/4131	o	-7
+-13.420170678011700	'~!(	135/136	'o	-6.6
+-12.436493246115600	)~!	143/144		-6.4
+-12.014917203874400	.)!(	2816/2835	.i	-6
+-11.031239771978200	)!(	891/896	i	-5
+-9.063884908185930	~!	2176/2187	s	-4.6
+-8.080207476289780	.!(	224/225	.n	-4.2
+-7.096530044393640	'~!	255/256	's	-4
+-6.534428654738690	!(	5103/5120	n	-3
+-4.567073790946400	)!	512/513	;	-2
+-2.740244274567840	.!	32768/32805	.h	-1.3
+-1.756566842671690	')!	1215/1216	';	-1
+-1.194465453016750
+1.194465453016750	.)|	1216/1215	.r	1
+1.756566842671690	'|	32805/32768	'	1.3
+2.740244274567840	)|	513/512	r	2
+4.567073790946400	|(	5120/5103	u	3
+6.534428654738690	.~|	256/255	.$	4
+7.096530044393640	'|(	225/224	'u	4.2
+8.080207476289780	~|	2187/2176	$	4.6
+9.063884908185930	)|(	896/891	*	5
+11.031239771978200	')|(	2835/2816	'*	6
+12.014917203874400	)~|	144/143		6.4
+12.436493246115600	.~|(	136/135	.e	6.6
+13.420170678011700	~|(	4131/4096	e	7
+15.387525541804000	|~	736/729	~	8
+17.354880405596300	~~|	99/98		9
+18.760133879733700	./|	2048/2025	./	10
+19.743811311629800	)|~	19683/19456		10.7
+20.305912701284800	/|	81/80	/	11
+22.694843607318300	.)/|	76/75		12
+23.116419649559500	'/|	531441/524288	'/	12.4
+23.955448448755500	)/|	41553/40960		13
+24.885981585695500	.|)	2097152/2066715	.f	13.5
+26.067451945247900	|)	64/63	f	14
+28.596908198695100	'|)	3645/3584	'f	15
+30.002161672832500	)|)	57/56		16
+30.985839104728600	.(|	56/55	.?	16.4
+31.407415146969900	|\	55/54	y	16.6
+32.391092578866000	(|	45927/45056	?	17
+34.358447442658300	'(|	50/49	'?	18
+35.118091464366500	~|)	49/48		18.4
+36.588490003137900	.(|(	8192/8019	.g	19
+37.309479738346700	'~|)	535815/524288		19.6
+37.871581128001700	/|~	46/45		19.9
+38.061940349778500	(|(	45/44	g	20
+39.698410644380200	~|\	16767/16384		20.8
+40.260512034035200	.//|	128/125	.=	21
+41.665765508172500	//|	6561/6400	=	22
+44.195221761619800	'//|	39/38	'=	23
+45.600475235757100	)//|	416/405		24

```

Comment:
unreduced

Reply:
Great work. Thank you. 

I guess we're just waiting for the paint to dry on those mina 28, 75, 93 and 140 changes.